job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml

```diff
@@ -94,8 +94,10 @@ paths:
         in: query
         type: string
         required: false
-        description: Used to specify the sort order, in the format `<field>:<direction>`.
-          Results can also be ordered by label by using `labels.<label-key>` as the field name.  The direction can be `asc` or `desc`.
+        description: |
+          Used to specify the sort order, in the format `<field>:<direction>`.
+          Results can also be ordered by label by using ```labels.<label-key>``` as the field name.
+          The direction can be `asc` or `desc`.
```

Comment:
Have you intentionally got multiple backticks

Reply:
single quote, single line working fine